src/EditorFeatures/TestUtilities/TestExtensionErrorHandler.cs

```diff
@@ -30,7 +31,14 @@ public void HandleError(object sender, Exception exception)
                 return;
             }
 
-            ExceptionUtilities.FailFast(exception);
+            RaiseFailFastException(IntPtr.Zero, IntPtr.Zero, 0);
+
+            // the RaiseFailFastException above may have not actually killed the process if a
+            // debugger was attached. Fall back to the CLR FailFast.
+            Environment.FailFast("FailFast was issued from devenv.exe.  A crash dump should be available.");
```

Comment:
You'll have to merge this with recent changes @jaredpar made. He removed the FailFast since that was making it hard to figure out which test was impacted.